data/tlg0732/tlg013/tlg0732.tlg013.1st1K-grc1.xml

```diff
@@ -0,0 +1,2699 @@
+                </respStmt>
+                <respStmt>
+                    <persName>Gregory Crane</persName>
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0732.tlg013.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Supplementum Aristotelicum</title>
+                            <editor>
+                                <persName>
+                                    <name>Ivo Bruns</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0732.tlg013.1st1K-grc1.xml">Alexander of Aphrodisias</author>
+                            <imprint>
+                                <publisher>Reimer</publisher>
+                                <pubPlace>Berlin</pubPlace>
+                                <date>1887</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
+                        </monogr>
+                        <ref target="http://www.archive.org/stream/p1supplementumar02akaduoft#page/116/mode/2up">Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+)" n="chapter"
```

Comment:
Some of the chapters are quite long. I think I would further subdivide them into sections, probably on the `<p>` tags, either by numbering the `<p>` tags or enclosing them in `<div>` elements.

Reply:
Again, there are only 38 paragraphs (not counting the toc) for 30 chapters, so I am not sure if this would help matters much.

@sonofmun

Reply:
The citation units here aren't quite as bad. But maybe we could do sentences here as well? At least then we have sense units that are small enough that if someone cites them you can actually find what they are talking about.

Reply:
It is OK for now to leave this with the citation scheme that it has. If people start reading or citing it and complaining, then we can change it